sources/packages/backend/libs/test-utils/src/factories/msfaa-number.ts

```diff
@@ -7,26 +7,76 @@ import {
 import { getISODateOnlyString } from "@sims/utilities";
 import * as faker from "faker";
 
-export function createFakeMSFAANumber(relations: {
-  student: Student;
-  referenceApplication?: Application;
-}): MSFAANumber {
+/**
+ * Represents the different MSFAA scenarios.
+ * More than one state can be combined, for instance,
+ * to create a signed MSFAA that was also cancelled.
+ */
+export enum MSFAAStates {
+  Pending = 0,
+  Signed = 1,
+  CancelledSystem = 2,
+  CancelledOtherProvince = 4,
```

Comment:
are we not using `3` due to bitwise operations?

Reply:
So, wouldn't it be good to comment here that a MSFAAStates 3 should be a "Signed" + "CancelledSystem"? That was my understanding at least.

Reply:
The combination between the enum items is free. It can be as below or others.
- Pending 
- Signed 
- Signed | CancelledSystem 
- Signed | CancelledSystem |  CancelledOtherProvince